src/components/ApplyDeveloper.tsx

```diff
@@ -145,6 +145,16 @@ const ApplyDeveloper: React.FC = () => {
               </Typography>
             </Box>
           </Fade>
+          <Fade>
+            <Box mb={5}>
+              <Typography variant="body1" gutterBottom className={classes.text}>
+                <i>
+                  All product teams will be working and meeting remotely. <br />
```

Comment:
Make this consistent with the other pages, if you change any of them.

Also, `<br />`? Is there a better way?